static/src/javascripts/projects/commercial/modules/dfp/display-lazy-ads.ts

```diff
@@ -1,11 +1,33 @@
+import { isInVariantSynchronous } from 'common/modules/experiments/ab';
+import { commercialGptLazyLoad } from 'common/modules/experiments/tests/commercial-gpt-lazy-load';
 import { partition } from '../../../../lib/partition';
 import { dfpEnv } from './dfp-env';
 import { enableLazyLoad } from './lazy-load';
 import { loadAdvert } from './load-advert';
 
+// For users in variant group of commercialGptLazyLoad,
+// these ad slots will be be lazy loaded rather than loaded instantly.
+// TODO - remove if GPT lazy loading is fully rolled out.
```

Comment:
Can we track differences in CLS _just_ for pages with "im" slots with the new lazy loading?

If we average out across all pages, I worry we might lose some CLS signal just for "im" ?

Reply:
Good call. I've asked Yen and Sara about this.

Reply:
With Sara we decided we'd collect core web vitals data for all users participating in the test in order to make sure that pages with an `im` slot don't see an increase in CLS.
We'll need to merge https://github.com/guardian/dotcom-rendering/pull/4288 before setting this test live.